### PR TITLE
Add linux/arm to the list of server builds

### PR DESCRIPTION
**Pull request by brendandburns**
@zmerlynn @roberthbailey

### CODE REVIEW

**Comment by roberthbailey** [comment]
lgtm

**Comment by brendandburns** [comment]
Rebased.

**Comment by davidopp** [comment]
@k8s-bot test this please

File: cluster/images/hyperkube/Makefile [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 
 all:
 	cp ../../saltbase/salt/helpers/safe_format_and_mount .
 	cp ../../saltbase/salt/generate-cert/make-ca-cert.sh  .
-	curl -O https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/amd64/hyperkube
+	curl -O https://storage.googleapis.com/kubernetes-release/release/${VERSION}/bin/linux/${ARCH}/hyperkube
 	sed -i "s/VERSION/${VERSION}/g" master-multi.json master.json
-	docker build -t gcr.io/google_containers/hyperkube:${VERSION} .
-	gcloud docker push gcr.io/google_containers/hyperkube:${VERSION}
+        sed -i "s/ARCH/${ARCH}/g" master-multi.json master.json
```
Comment (davidopp) [inline cluster/images/hyperkube/Makefile]: there's an indentation issue here (offset by one space) but you can fix it in a followup PR

**Comment by davidopp** [comment]
LGTM

**Comment by luxas** [comment]
@brendandburns Thanks for this contribution! I am running k8s on my ARM devices, here is my project, if you haven't noticed: https://github.com/luxas/kubernetes-on-arm
I have been running k8s on ARM since `v1.0.1`

Is there a thread for ARM? Otherwise I may create one.

BTW, could you take a look at https://github.com/kubernetes/kubernetes/pull/17213#issuecomment-159821712?

**Comment by saturnism** [comment]
@luxas btw i demoed a kubernetes cluster on raspi - your repo rocked! :)

**Comment by luxas** [comment]
Thanks! Now I've released `v0.6.0` of `kubernetes-on-arm`, welcome to discuss it in issues there!

**Pull request merged**